src/Resources/Users/Toyboxs.ts

```diff
@@ -3,8 +3,8 @@ import AbstractResource from '../AbstractResource';
 /**
  * _Updating a user's toybox_
  *
- * - `$whirli.user.toybox.designateToyForReturn(stockId);`
- * - `$whirli.user.toybox.undesignateToyForReturn(stockId);`
+ * - `client.user.toybox.designateToyForReturn(stockId);`
+ * - `client.user.toybox.undesignateToyForReturn(stockId);`
```

Comment:
Think this comment should be `client.users.toybox`.

This is what I mean by these comments not being maintainable. I would suggest removing them.